lib/calendarium-romanum/cli.rb

```diff
@@ -115,5 +107,25 @@ def die!(message, code=1)
       STDERR.puts message
       exit code
     end
+
+    private
```

Comment:
No need for `private` here, we are already in a block of private methods. See a few lines above.

Reply:
Done. Just posting it here as a checklist for myself.